pkg/scheduler/framework/session_plugins.go

```diff
@@ -403,6 +403,7 @@ func (ssn *Session) JobEnqueueable(obj interface{}) bool {
 			}
 			if res > 0 {
 				hasFound = true
+				break
```

Comment:
Can not break here. Need loop all fn in each plugin to make sure there is no reject (res <0).
Do you hit some issue in your env?

Reply:
i didn't hit any issue in my env,i am ignoring this,i will close this pr,thanks :)